src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.cs

```diff
@@ -54,9 +43,17 @@ public static T GetDeclarationSyntax<T>( this ISymbol symbol )
 			return decl;
 		}
 
-		public static bool IsFromOtherAssembly( this ITypeSymbol type, Compilation compilation ) {
-			return type.ContainingAssembly != compilation.Assembly;
-		}
+		#region Stuff that can be deleted with  ImmutableGenericArgumentAnalyzer

```

Comment:
@omsmith did "find all references" on the things in this file. This stuff I'll leave to you to delete once you're done with `ImmtuableGenericArgumentAnalyzer`